sunpy/net/jsoc/attrs.py

```diff
@@ -94,6 +94,16 @@ def __or__(self, other):
     __ror__ = __or__
 
 
+class PrimaryKey(_VSOSimpleAttr):
+    """
+    Extra key,values pair produced by the user to query jsoc
+    """
+    def __init__(self, key, value):
+        Attr.__init__(self)
+        self.__class__.__name__ += '_{}'.format(key)
+        self.value = value
+
```

Comment:
@Cadair do you have any idea how to avoid this:
```python
>>> a = jsoc.attrs.PrimaryKey('HARPNUM',638)
>>> a
 <PrimaryKey_HARPNUM(638)>
>>> a.value
 638
>>> b = jsoc.attrs.PrimaryKey('HARPNUM',12)
>>> a
 <PrimaryKey_HARPNUM_HARPNUM(638)>
>>> b
 <PrimaryKey_HARPNUM_HARPNUM(12)>
```
So, what's happening here is that each time I call that `PrimaryKey` I get not a new `PrimaryKey_HARPNUM` but it adds to the previous one... well, I get a new one (as you can see b value changed), but the names are changed to them all...